dsl/procedures/importMicroservices/procedure.dsl

```diff
@@ -0,0 +1,15 @@
+import java.io.File
+
+procedure 'Import Microservices',
+	description: 'Import microcervices using Docker Compose file and creates corresponding application models for them in ElectricFlow', {
+
```

Comment:
Most groovy based procedures need a 'setup' step that retrieves the groovy dependency jars. The actual logic for retrieval is already implemented in the 'Setup' procedure in this plugin. So, you just need to add it as the first step here.
See https://github.com/electric-cloud/EC-Docker/blob/master/dsl/procedures/deployService/procedure.dsl#L6 for reference.

Reply:
Done

Reply:
Done